repos/msbuild.proj

```diff
@@ -1,10 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+
+  <!--ItemGroup>
+    <EnvironmentVariables Include="BUILD_BUILDNUMBER=20180416.01" />
+  </ItemGroup-->
   <PropertyGroup>
-    <LatestCommit>71eedf98975925fa7f2f3e669d19ba6c81391048</LatestCommit>
+    <LatestCommit>d7690fb9d3d7b97917243ae63aed97dc3d146e85</LatestCommit>
     <PackagesOutput>$(ProjectDirectory)/artifacts/$(Configuration)/packages</PackagesOutput>
-    <BuildCommand>$(ProjectDirectory)build/build$(ShellExtension) build -DotNetBuildFromSource -DotNetCoreSdkDir $(DotNetCliToolDir) -bootstraponly -skiptests -pack -configuration $(Configuration) /p:DisableNerdbankVersioning=true /p:GitHeadSha=$(LatestCommit)</BuildCommand>
+    <OutputVersionArgs>/p:VersionBase=15.6.82 /p:DisableNerdbankVersioning=true /p:PreReleaseVersionLabel="preview2" /p:BUILD_BUILDNUMBER=$([System.DateTime]::Now.ToString("yyyyMMdd")).01</OutputVersionArgs>
```

Comment:
For repeatability, `/p:BUILD_BUILDNUMBER=$([System.DateTime]::Now.ToString("yyyyMMdd")).01` makes me a little nervous. Last I saw, the Core repos base the version number on the date of the last commit, to make this deterministic.

For now, just having a date override that we could pass in later would work for me.

Reply:
OK.  I added LatestCommitDate and used that.